android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -654,6 +652,36 @@ public void pickFeature(float posX, float posY) {
     }
 
     /**
+     * Adds a {@link Marker} to the map which can be used to dynamically add points and polylines
+     * to the map.
+     * @return Newly created {@link Marker} object.
+     */
+    public Marker addMarker() {
+        checkPointer(mapPointer);
+        long markerId = nativeMarkerAdd(mapPointer);
+        return new Marker(mapView.getContext(), markerId, this);
+    }
+
+    /**
+     * Removes the passed in {@link Marker} from the map.
+     * @param marker to remove from the map.
+     * @return whether or not the marker was removed
+     */
+    public boolean removeMarker(Marker marker) {
+        checkPointer(mapPointer);
+        checkId(marker.getMarkerId());
+        return nativeMarkerRemove(mapPointer, marker.getMarkerId());
```

Comment:
does it make sense to set the `markerID` of this `marker` as `0` if the `nativeMarkerRemove` succeeds?